### ISSUE TITLE
Typescript declarations

### ISSUE THREAD

**Issue by chechoreyes** [staeco/mauro#5]
Hello,

This package has declarations for typescript? If not, please make 👏

**Comment by yocontra** [comment]
It does not - I would like to rewrite it in typescript eventually, if you'd like to help let me know!

**Comment by chechoreyes** [comment]
Yes, i'd like to help, send me an email, to sergio.reyes.gajardo@gmail.com.

Regards!
